quantum/quantum.c

```diff
@@ -59,6 +59,10 @@ extern backlight_config_t backlight_config;
 #include "encoder.h"
 #endif
 
+#ifdef OLED_DRIVER_ENABLE
```

Comment:
This should be moved to `quantum.h` and removed from this file.

Reply:
Fixed